### PR TITLE
edited warning

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/5621

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @nam31, for your information

Thank you

File: windows/security/identity-protection/user-account-control/how-user-account-control-works.md [hunk 1]
```diff
@@ -278,7 +278,7 @@ The slider will never turn UAC completely off. If you set it to <b>Never notify<
 
 > **Important:**  In order to fully disable UAC you must disable the policy **User Account Control: Run all administrators in Admin Approval Mode**.
 > 
-> **Warning:**  Universal Windows apps will not work when UAC is disabled.
+> **Warning:**  Some Universal Windows apps may not work when UAC is disabled.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/user-account-control/how-user-account-control-works.md:281]:
```suggestion
> **Warning:**  Some Universal Windows Platform apps may not work when UAC is disabled.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by nam31** [comment]
@Dansimp PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by nam31** [comment]
@DulceMontemayor PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by illfated — APPROVED** [review]

**Review by DulceMontemayor — APPROVED** [review]

**Pull request merged**